docs/developing.rst

```diff
@@ -535,21 +545,30 @@ Example::
     from toil.job import Job
 
     class HelloWorld(Job):
-        def __init__(self, message):
+        def __init__(self, inputFileID):
             Job.__init__(self,  memory="2G", cores=2, disk="3G")
-            self.message = message
+            self.inputFileID = inputFileID
+
+        with fileStore.readGlobalFileStream(self.inputFileID) as fi:
+            with fileStore.writeGlobalFileStream() as (fo, outputFileID):
+                fo.write(fi.read() + 'World!')
+            return outputFileID
 
-        def run(self, fileStore):
-            return "Hello, world!, here's a message: %s" % self.message
 
     if __name__=="__main__":
         options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
         options.logLevel = "INFO"
-        job = HelloWorld("enter the void empty and become wind")
+
+
         with Toil(options) as toil:
-            jobStoreFileID = toil.fileStore(job).importFile('s3://somebucketname/anditskey')
-            toil.exportFile(jobStoreFileID, 'file:///some/local/path')
-            toil.run(job)
+            if not toil.options.restart:
+                inputFileID = toil.importFile('file:///some/local/path')
+                outputFileID = toil.start(HelloWorld(inputFileID))
+            else:
+                # files should be restaged here
```

Comment:
Makes no sense. Files are already stage and don't need to be restaged (as in staged again)